gcloud/monitoring/client.py

```diff
@@ -195,6 +203,126 @@ def metric_descriptor(self, type_,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
+
+             >>> type_ = 'compute.googleapis.com/instance/cpu/utilization'
+             >>> metric = client.metric(type_, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        For custom metrics, the simplest way to get the type is from the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` class used to
+        create the custom metric::
+
+             >>> metric = client.metric(metric_descriptor.type, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        :type type_: string
+        :param type_: The metric type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated :class:`MetricDescriptor`.
+
+        :rtype: :class:`~gcloud.monitoring.metric.Metric`
+        :returns: The Metric created with the passed in arguments
+        """
+        return Metric(type=type_, labels=labels)
+
+    @staticmethod
+    def resource(type_, labels):
+        """Construct a resource object.
+
+        For a list of possible MonitoredResources and their associated labels,
+        see:
+
+        https://cloud.google.com/monitoring/api/resources
+
+        For example, to create a Resource object for a GCE instance::
+
+            >>> resource = client.resource('gce_instance', labels= {
+            ...     'project_id': 'my-project-id',
+            ...     'instance_id': 'my-instance-id',
+            ...     'zone': 'us-central1-f'
+            ... })
+
+        :type type_: string
+        :param type_: The monitored resource type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated
+                       :class:`ResourceDescriptor`.
+
+        :rtype: :class:`gcloud.resource.Resource`
+        :returns: The Resource created with the passed in arguments
+        """
+        return Resource(type_, labels)
+
+    @staticmethod
+    def timeseries(metric, resource, value,
+                   start_time=None, end_time=None):
+        """Constructs a TimeSeries object for a single data point.
+
+        .. note::
+
+           While TimeSeries objects returned by the API can have
+           multiple data points when queried as aggregates, TimeSeries created
+           on the client-side can have at most one point.
+
+        For example::
+
+            >>> timeseries = client.timeseries(metric, resource, value,
+            ...     end_time=end)
```

Comment:
I find the rules about where to indent multiline function calls slightly confusing :

```
client.timeseries(value, labels={
    'example': 3
})
```
inline dicts as arguments seem to get wrapped to start of function call, right?

```
client.timeseries(metric, resource,
                            end_time=end)
```

new params get wraped to the next param to start of params, right?

If those are both correct, what would you do if you had an inline dict then more arguments?

```
client.timeseries(value, labels={
    'example': 3
}, end_time=end)
```
where does end_time go in that last example?